src/nifreeze/data/base.py

```diff
@@ -99,12 +99,12 @@ def __len__(self) -> int:
 
         return self.dataobj.shape[-1]
 
-    def _getextra(self, idx: int | slice | tuple | np.ndarray) -> tuple[Unpack[Ts]]:
-        return ()  # type: ignore[return-value]
+    def _getextra(self, idx: int | slice | tuple | np.ndarray) -> np.ndarray | None:
```

Comment:
Why changing the tuple? The idea here was that we can add extra objects if necessary in the future